src/Package/MSBuild.VSSetup/MSBuild.VSSetup.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFramework>$(FullFrameworkTFM)</TargetFramework>
     <!-- VS Insertion -->
     <TargetVsixContainerName>Microsoft.Build.vsix</TargetVsixContainerName>
-    <VisualStudioInsertionComponent>Microsoft.Build</VisualStudioInsertionComponent>
+    <VisualStudioInsertionComponent Condition="'$(MonoBuild)' != 'true'">Microsoft.Build</VisualStudioInsertionComponent>
```

Comment:
What problem does this cause? is it not avoided by `ShouldSkipProject`?

Reply:
You are correct, 👍 . Removed.